synapseclient/models/mixins/storable_container.py

```diff
@@ -1015,3 +1428,65 @@ def _resolve_sync_from_synapse_result(
                 )
                 if failure_strategy == FailureStrategy.RAISE_EXCEPTION:
                     raise exception
+
+    def _format_entity_info_for_tree(
```

Comment:
Nice, we'll have to add the recordSet entity soon to this list!

Reply:
The logic here should work without adding it to the list, it'll just have the ? icon though until we do add it to this list